src/ansys/fluent/core/services/interceptors.py

```diff
@@ -109,7 +110,8 @@ def _intercept_call(
             grpc_ex = response.exception()
             ex = RuntimeError(grpc_ex.details())
             ex.__context__ = grpc_ex
-            raise ex from None
+            if not _license_error(ex):
+                raise ex from None
```

Comment:
@seanpearsonuk We have added `_license_error()` here to avoid it's imports in each of `standalone_launcher.py`, `container_launcher.py` and `pim_launcher.py`.